tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -700,7 +700,7 @@ public function testInventoryRequest() {
       $logEntryFound = 0;
       $rows = $log->find("`direction` = 'O' AND `id` > '$lastLogId'");
       foreach ($rows as $row) {
-         if ($row['topic'] == $agent->getTopic() . '/Command/Inventory') {
+         if ($row['topic'] == 'Command/Inventory') {
```

Comment:
without this PR the tests select the log entry for the tested agent. Now, the topic does not contains this information. I think the unit  test is degraded by your change. We should add some assertions to ensure we will not wrongly pick a message for an other item.

Same comment for several other changes below in unit tests.